crates/swc_css_modules/src/hash.rs

```diff
@@ -0,0 +1,40 @@
+use std::{
+    collections::hash_map::DefaultHasher,
+    hash::{Hash, Hasher},
+    path::Path,
+};
+
+pub struct StyleHash;
+
+impl StyleHash {
+    ///
+    /// Generated content_hash from article content
+    fn generate_hash_by_content(content: &str) -> String {
+        let mut hasher = DefaultHasher::new();
+        content.hash(&mut hasher);
+        hasher.finish().to_string()
+    }
+
+    ///
+    /// Generate the prefix corresponding to css_modules according to the css
+    /// file path and file content
+    pub fn generate_css_module_hash(abs_filepath: &str, content: &str) -> String {
```

Comment:
We should not use `content` - it is bad for perf and hash should not be related to content (if I add `color: red` declaration we should not change hash)